_posts/2021-02-01-coalesce-dbt-summary.markdown

```diff
@@ -0,0 +1,288 @@
+
+In two very different talks, both Tristan Handy and Andrea Kopitz teach us that accepted knowledge is effectively the building of organisational consensus. Andrea's comment around creating some documentation - even if only temporarily useful - is often better than no documentation at all. It re-iterated to me also the importance of being assertive and descriptive in documentation to allow others to challenge, so that together we may reach a better consensus.
+
+On a more technical level, I took away from many talks the success teams had encountered by consolidating the knowledge and business logic of data domain and insight generation within their DBT projects, and how this had promoted more transparency and trust between consumers and producers / curators of data.
+
+
+### The Right Testing Matters
+
+There were a couple of great talks about testing in Coalesce 2020; beyond introducing tools and technologies, they more importantly offered:
+
+* The importance of testing as part of [SRE](https://sre.google/sre-book/table-of-contents/) obligations and meeting service level agreements
+* Different types of testing, e.g. modelling regression vs. data regression, including periodic factors
+* Comparing the value of testing against the cost of troubleshooting queries from consumers
+* The use of testing as a form of documentation about expectations and subject matter expertise
+
+One important reminder I took away was, ensure to test the right thing: avoid vanity tests and appreciate that needs and expectations, as well as data, change over time - and therefore, so should your tests; even if it means deprecation.
+
+
+### Trust 
+
+Perhaps the most enduring theme throughout the presentations was trust:
+
+* Why is it important?
+* What happens when we don't have it?
+* How can we build it?
+* How can we retain it?
+
+Trust exists in many contexts; some examples were how teams build trust amongst themselves with testing and documentation; how they maintain it with peer teams using SLAs, relationships and communication, and good engineering practice; how they build it with stakeholders and decision makers, to boost their standing and perception as a critical partner within an organisation.
+
+I sensed in many presentations the burden within data teams that their work should reflect the most true view possible of their organisation. The concept of trust featured heavily in many of their endeavours. All of the following stood out as important assets in building and maintaining that trust.
```

Comment:
I'm not sure what this para means. "the burden within data teams..."  ?
You mean that they alone shoulder this burden? 
That it's their responsibility to ensure the _data_ is true? 

"the most true view of their organisation" is an odd phrase.  "the most accurate view"? 

I'm not sure what you're shooting at...
"that the data represents the most accurate portrayal of their organisation" ?

Reply:
it has multiple aspects to which people likely can relate
- we didn't mess your data up when we moved it somewhere else
- if the data is messed up, we will already know about it and be dealing with it
- we modelled the data consistently based on the agreed definitions the business helped us define
etc. etc.

accurate is still a subjective word. "unfettered" is probably more on the mark.